rebuild-build-single.js

```diff
@@ -82,43 +48,69 @@ function deleteFrameworkFiles(frameworkPath, filesToDelete) {
 
 /**
  * @param {string} framework
+ * @param {boolean} useCi
  */
-function rebuildFramework(framework) {
+function rebuildFramework(framework, useCi) {
   const components = framework.split("/");
 
   if (components.length !== 2) {
-    console.log(
-      `ERROR: invalid name ${framework}. It must contain exactly one /.`
-    );
+    console.log(`ERROR: invalid name ${framework}. It must contain exactly one /.`);
     process.exit(1);
   }
 
   const [keyed, name] = components;
   const frameworkPath = path.join("frameworks", keyed, name);
 
+  const filesToDelete = [
+    "yarn-lock",
+    "dist",
+    "elm-stuff",
+    "bower_components",
+    "node_modules",
+    "output",
+    !useCi && "package-lock.json",
+  ].filter(Boolean);
+
```

Comment:
I'll have to do a manual merge. I didn't like that before and I still don't want this style in here.